deny.toml

```diff
@@ -86,5 +86,5 @@ unsound = "warn"
 yanked = "deny"
 notice = "warn"
 ignore = [
+    "RUSTSEC-2023-0071"
```

Comment:
A comment why would be nice

Reply:
It was a backport of a change from [another PR](https://github.com/kamu-data/kamu-cli/pull/665) (merged) but I've added context as to why the change was made